src/__init__.py

```
@@ -1 +1,15 @@
-# Source package
\ No newline at end of file
+# Source package
+
+# Compatibility shim for tests expecting MemoryStorage.key_builder
+try:
+    from aiogram.fsm.storage.memory import MemoryStorage
+    from aiogram.fsm.storage.base import StorageKey
+
+    if not hasattr(MemoryStorage, "key_builder"):
+        def _key_builder(self, *, bot_id: int, chat_id: int, user_id: int) -> StorageKey:
```

Ruff (0.12.2)

Unused function argument: `self`

(ARG001)